homeassistant/components/homekit/type_thermostats.py

```diff
@@ -839,8 +839,7 @@ def _get_temperature_range_from_state(
     # the max to appears to work, but less than 0 causes
     # a crash on the home app
     min_temp = max(min_temp, 0)
-    if min_temp > max_temp:
-        max_temp = min_temp
+    max_temp = max(max_temp, min_temp)
```

Comment:
min and max are significantly slower if they are in a hot path.  Likely doesn't matter in this case, though.  I don't think they are well optimized for replacing single comparisons, but they are good idea for long sets of values.

Reply:
The change is related to the `consider-using-max-builtin` check. I would be fine with disabling it completely if we want to do that. https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/consider-using-max-builtin.html#consider-using-max-builtin-r1731

Reply:
I think in the most cases, it's the right thing to do, it's rare, that we have such performance sensitive code that uses this that we would have to add more than one or two, so I'm fine with it as is